src/main/java/jimmy/mcgymmy/model/food/Macronutrient.java

```diff
@@ -4,12 +4,11 @@
 import jimmy.mcgymmy.commons.util.CollectionUtil;
 
 public abstract class Macronutrient {
-    public static final String MESSAGE_CONSTRAINTS = "values should only contain non-negative integer values.";
+    public static final String MESSAGE_CONSTRAINTS = " amount can only take in value larger than 0 and less than 1000";
```

Comment:
Maybe `Amount can only take in values between 0 and 1000`. I believe `0` is still acceptable as it is also the default value. I'm not too sure atm whether our upper bound is `1000` or `999`.

Reply:
Currently it is 999 iirc as the regex checks for the presences of between 1 to 3 numbers

Reply:
also 0 is valid right? can we change it to `can only contain non-negative integers less than 1000`

Reply:
For the current change it is ```amount can only take in value larger than 0 and less than 1000```

Reply:
Sure